docs/user/management.asciidoc

```diff
@@ -92,6 +92,10 @@ Create and manage the index patterns that help you retrieve your data from {es}.
 Monitor the generation of reports&mdash;PDF, PNG, and CSV&mdash;and download reports that you previously generated.
 A report can contain a dashboard, visualization, saved search, or Canvas workpad.
 
+| <<managing-alerts-and-actions, *Alerts and Actions*>>
```

Comment:
Should this section move above `Index Patterns` to keep alphabetical order?

Reply:
++ I have updated to restore alphabetical order